### PR TITLE
Soc 2418

### PR DESCRIPTION
**Pull request by qaga**
refactored discussion tests
most of them should run on wiki dedicated for discussions automated tests

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/discussions/Promoting.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/discussions/Promoting.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.contentpatterns.MercuryWikis'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryWikis.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
   public static final String MLP = "mlp";
   public static final String MEDIAWIKI_119 = "mediawiki119";
   public static final String DE_WIKI = "de.demercurytests";
+  public static final String DISCUSSIONS_AUTO = "DAUTO";
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryWikis.java]: wiki name should be lower case

Comment (qaga) [inline src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryWikis.java]: done

**Pull request merged**